### DISCUSSION TITLE
Remove alignment in "fix common errors"

### DISCUSSION THREAD

**Discussion by filip11111** [SubtitleEdit/subtitleedit#7181] [Ideas]
Hi, I have a suggestion, if you could put optional choice to remove alignment to "fix common errors" window? Otherwise I have to do it with each subtitle individually since my TV does not support {\an8} characters in srt. Thank you.

**Comment by ivandrofly** [comment]
That is a good idea, maybe adding a clear alignments option from the context menu!

But for now you can use *Multiple replace* windows to do that the removing.

Edit => Multiple Replace

![image](https://github.com/SubtitleEdit/subtitleedit/assets/6579818/21efaae3-9223-4c7e-a422-fcb2af70c97a)

**Comment by ivandrofly** [comment]
> Hi, I have a suggestion, if you could put optional choice to remove alignment to "fix common errors" window? Otherwise I have to do it with each subtitle individually since my TV does not support {\an8} characters in srt. Thank you.

I just found that the feature is already there, you don't even need to use the "Multiple replace" windows
just select all the lines and right click with the mouse there is a option "Remove formatting"click there and you will see the "Remove alignment"

**Comment by niksedk** [comment]
Latest beta has improved "Remove formatting" (first fix item) in Tools - Batch convert
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.13/SubtitleEditBeta.zip
Let me know how it works

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/71b72782-a77d-4673-a51b-ffd40a2bc113)

**Reply by mzso** [reply]
@niksedk
Is this not available in the main menubar? I would expect it in the tools menu.
It seems rather arbitrary what appears in the tools menu (remove hearing impaired text), in the context menu (remove formatting) or both (fix common errors).
